packages/dds/tree/src/test/simple-tree/unhydratedNode.spec.ts

```diff
@@ -177,6 +177,24 @@ describe("Unhydrated nodes", () => {
 		assert.equal(Tree.status(object), TreeStatus.New);
 	});
 
+	it("preserve events after hydration - minimal", () => {
+		const log: unknown[] = [];
+		const leafObject = new TestLeaf({ value: "value" });
+
+		Tree.on(leafObject, "nodeChanged", (data) => {
+			log.push(data);
+		});
+		Tree.on(leafObject, "treeChanged", () => {
+			log.push("treeChanged");
+		});
+
+		hydrate(TestLeaf, leafObject);
+		leafObject.value = "new value";
+		// Assert that the event fired
+		// TODO: Eventually the order of events should be documents, and an approach like this can test that they are ordered as documented.
```

Comment:
```suggestion
		// TODO: Eventually the order of events should be documents, and an approach like this can test that they are ordered as documented.
```
```suggestion
		// TODO: Eventually the order of events should be documented, and an approach like this can test that they are ordered as documented.
```